src/parser/jobs/brd/EightyBoi.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * @author Yumiya
+ */
+
+import ACTIONS from 'data/ACTIONS'
+import Module from 'parser/core/Module'
+
+const SETUP_TIME = 3800
+const SONG_MAX_DURATION = 30000
+const DOT_INTERVAL = 3000
+const ANIMATION_LOCK = 700
+
+// For easy access
+const THE_WANDERERS_MINUET = 0
+const MAGES_BALLAD = 1
+const ARMYS_PAEON = 2
+
+export default class SongUsage extends Module {
+	static handle = 'eightyboi'
+	static dependencies = [
+		'invuln',
+		'downtime',
+	]
+
+	_debugEvents = []
+
+	_lastDotTickTimestamp = 0
+	_eightyBoiCasts = []
+	_songList = [
```

Comment:
With all your classes elsewhere I'm surprised this isn't following suit. These could def be moved into a more generic structure. DRY and all that.